### PR TITLE
Icon script automatate build

### PR DESCRIPTION
**Pull request by agliga**
## Description
Changed icon build script to rerun build after the files are written

## Context
Change it so the build runs after all the files are written.

### CODE REVIEW

File: scripts/generate-images.js [hunk 1]
```diff
@@ -9,26 +9,44 @@ const config = require('./image-config.json');
...
-    await writeSymbols(result, dsVersion);
-  } catch (e) {
-    console.error('An error has occurred', e)
-  }
-});
+async function run() {
+  await Promise.all(files.map(async (filePath) => {
+    try {
+      const dsVersion = filePath.indexOf('ds6') > -1 ? 'ds6' : 'ds4';
+      const data = await fs.promises.readFile(filePath, 'utf8')
```
Comment (msendlakowski) [inline scripts/generate-images.js]: can you add a semi-colon

File: scripts/generate-images.js [hunk 2]
```diff
@@ -9,26 +9,44 @@ const config = require('./image-config.json');
...
+      const result = await svgo.optimize(data, { path: filePath });
+      await fs.promises.writeFile(result.path, result.data)
+      const svgGenerator = new SVGGenerator(result, dsVersion)
+      await svgGenerator.generateAllBase64();
+      await writeSymbols(result, dsVersion);
+      console.log(`Wrote all ${dsVersion} files`);
+    } catch (e) {
+      console.error('An error has occurred', e)
```
Comment (msendlakowski) [inline scripts/generate-images.js]: can you add a semi-colon

File: scripts/generate-images.js (hunk 1 above)
Comment (senthilp) [reply scripts/generate-images.js]: Should this not be caught by ESlint?

Comment (msendlakowski) [reply scripts/generate-images.js]: There is so little JS in skin that we don't have eslint running.  And there isn't any JS published to users. 
I'll put an issue in the backlog for Ian and the team to consider, now that we're automating more of the repo.

Comment (agliga) [reply scripts/generate-images.js]: We don't have an eslint setup in skin :(

**Pull request merged**